package.json

```diff
@@ -25,12 +25,13 @@
     "configstore": "^6.0.0",
     "cookiefile": "^1.0.10",
     "date-fns": "^2.9.0",
-    "graphql": "^16.3.0",
```

Comment:
I thought `graphql-request` might require `graphql` but I guess not?

Reply:
Yeah… I found that quite surprising myself. I made the change expected requests to start failing… they never did 🥹